coq-makefile: print-pretty-timed-diff broken

When trying to use the print-pretty-timed-diff target, there is some problem with the invoked python code (at least when using Python 3.7.2):

```
> make -f Makefile.coq print-pretty-timed-diff 
Traceback (most recent call last):
  File "/home/fakusb/.opam/coq-dev/lib/coq//tools/make-both-time-files.py", line 16, in <module>
    print_or_write_table(table, args[3:])
  File "/home/fakusb/.opam/coq-dev/lib/coq/tools/TimeFileMaker.py", line 215, in print_or_write_table
    print(table.encode("utf-8"), file=binary_stdout)
TypeError: a bytes-like object is required, not 'str'
make: *** [Makefile.coq:337: print-pretty-timed-diff] Error 1
```

this occurs both on coq 8.8.2 and master (at least in a build from 2 weeks ago)

--- a/tools/TimeFileMaker.py
+++ b/tools/TimeFileMaker.py
@@ -209,11 +209,10 @@ def make_table_string(times_dict,
 
 def print_or_write_table(table, files):
     if len(files) == 0 or '-' in files:
-        try:
-            binary_stdout = sys.stdout.buffer
-        except AttributeError:
-            binary_stdout = sys.stdout
-        print(table.encode("utf-8"), file=binary_stdout)
+        if hasattr(sys.stdout, 'buffer'):
+            sys.stdout.buffer.write(table.encode("utf-8"))
+        else:
+            sys.stdout.write(table)
     for file_name in files:
         if file_name != '-':
             with open(file_name, 'w', encoding="utf-8") as f:

